libimage/manifest_list.go

```diff
@@ -547,7 +548,7 @@ func (m *ManifestList) AnnotateInstance(d digest.Digest, options *ManifestListAn
 		}
 	}
 	if options.Subject != "" {
-		ref, err := m.parseNameToExtantReference(ctx, options.Subject, true, "subject for image index")
+		ref, err := m.parseNameToExtantReference(ctx, options.Subject, true, "subject for image index", nil)
```

Comment:
I think these are eventually going to need a `SystemContext` as well, though that’s probably not this PR.

Reply:
The artifact codepath isn't plumbed for TLS verification at all so I agree that's probably a separate PR.